### DISCUSSION TITLE
下划线识别不了

### DISCUSSION THREAD

**Discussion by Dkaaaaa** [RapidAI/TableStructureRec#73] [General]
有没有什么参数或者方法

**Comment by Joker1212** [comment]
emmm,可能得调节ocr的参数了，可以去RapidOCR看看参数，或者切换另外的ocr模型

**Comment by Joker1212** [comment]
文字下面的下划线一般是识别不了的，因为它是单个字符在识别，单独的下划线可能识别为-

**Reply by Dkaaaaa** [reply]
你好，非常感谢答复，那是否有类似tesseract中设置白名单的操作呢，就是只识别我指定的字符，如果源码有这个参数可以设置，是否可以告知具体的位置，谢谢！

**Reply by Joker1212** [reply]
暂时没有哈，这个的话，可以去rapidOCR提个Issue，或者最简单的办法是，直接过滤一下RapidOCR的输出，因为表格识别是支持入参传入ocr_result的，也支持内部不执行ocr，只输出表格的物理box和行列逻辑坐标

**Reply by Dkaaaaa** [reply]
非常感谢！
